README.md

```diff
@@ -1,4 +1,4 @@
-![](./js/assets/logos/blue.png)
+<img src="./js/assets/logos/blue.png" alt="Logo" width=200>
```

Comment:
`yellow.png` plz.
Also, html should not be used here. Resize the logo manually.